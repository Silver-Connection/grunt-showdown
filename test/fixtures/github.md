This is a ~~striked string~~.