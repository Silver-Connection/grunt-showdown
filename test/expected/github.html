<p>This is a <del>striked string</del>.</p>